packages/core/scripts/creator-umip/1_Propose.js

```diff
@@ -34,7 +46,7 @@ async function runExport() {
         data: addCreatorToRegistryTx
       }
     ],
-    { from: proposerWallet, gas: 2000000 }
+    { from: proposerWallet, gas: 2000000, gasPrice: gasEstimator.getCurrentFastPrice() }
```

Comment:
Same question as before: do you need the `gas`?

Reply:
I think we originally did this because ganache-fork blows up sometimes when asked to estimate gas, so I just left it as it was.